pkg/driver/camera/camera_linux.go

```diff
@@ -79,6 +79,7 @@ func init() {
 // Initialize finds and registers camera devices. This is part of an experimental API.
 func Initialize() {
 	discovered := make(map[string]struct{})
+	discover(discovered, "/dev/v4l/by-id/*")
```

Comment:
Can you write a test for this and also describe how you've manually tested this?